### PR TITLE
Rename to clarify purpose of document

### PR DESCRIPTION
**Pull request by yanokwa**
* Rename current README to SOP and clean up title
* Add new README which is a simple landing page

Easiest review path is to step through the commits.

### CODE REVIEW

File: TSC-1/README.md [hunk 1]
```diff
@@ -1,128 +1,7 @@
-# ODK TSC Landing Page
+# TSC 1
 
-The Technical Steering Committee is the technical governing body for Open Data Kit 1 core projects.
+TSC 1 is responsible for ODK Collect, ODK Aggregate, ODK XLSForm, ODK Build, ODK Briefcase, ODK Central, ODK JavaRosa, and the ODK XForms specification.
```
Comment (smoyth) [inline TSC-1/README.md]: Is this going to be the authoritative list of these? I hope so!

Comment (yanokwa) [reply TSC-1/README.md]: It has to be other places (e.g., http://opendatakit.org/community/governance/), but this can be the authoritative list.

Comment (smoyth) [reply TSC-1/README.md]: But it's also in the main TSC document... Can we convert this to just refer to that?

Comment (yanokwa) [reply TSC-1/README.md]: Yeah, I see that now. Working on it...

**Pull request merged**